cf-custom-resources/lib/alb-rule-priority-generator.js

```diff
@@ -88,8 +88,8 @@ const calculateNextRulePriority = async function (listenerArn) {
     if (rules.length > 0) {
         // Take the max rule priority, and add 1 to it.
         const rulePriorities = rules.map(rule => {
-            if (rule.Priority === "default") {
-                // We treat the default rule as having priority 0
+            if (rule.Priority === "default" || rule.Priority === '50000') {
+                // We treat the default rule, and the max rule as having priority 0
```

Comment:
What do you think of this:
```js
// priorityForRootRule is the max priority number that's always set for the listener rule that matches the root path "/"
const priorityForRootRule = "50000"
```
and this comment becomes:
```js
// Ignore the root rule's priority since it has to be always the max value.
```